js/modules/k6/http/http_request.go

```diff
@@ -425,9 +426,16 @@ func (h *HTTP) request(ctx context.Context, preq *parsedHTTPRequest) (*HTTPRespo
 		}
 	}
 
+	var transport http.RoundTripper = netext.NewTransport(state.Transport, state.Samples, &state.Options, tags)
```

Comment:
See other note - probably 2 variables are needed - one `http.RoundTripper` that would be injected to the `http.Client` and can either be the `netext.Transport` or the `ntlmssp.Negotiator`, but we should also save the original result from `netext.NewTransport()` so we can call the `GetTrail()` method.